crates/router/src/connector/authorizedotnet.rs

```diff
@@ -603,23 +656,33 @@ fn get_error_response(
         .parse_struct("AuthorizedotnetPaymentsResponse")
         .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
 
-    logger::info!(response=?response);
-
-    Ok(response
-        .transaction_response
-        .errors
-        .and_then(|errors| {
-            errors.into_iter().next().map(|error| types::ErrorResponse {
-                code: error.error_code,
-                message: error.error_text,
+    match response.transaction_response {
+        Some(transaction_response) => Ok({
```

Comment:
get_error_response will be called at 4xx or 5xx, do we still need to handle the response as `AuthorizedotnetPaymentsResponse`? if we can replicate this scenario please confirm whether we are receiving `AuthorizedotnetPaymentsResponse` type, if not we can have separate struct for error

Reply:
This connector gives 200 status for all error cases .the error response is in same format of `AuthorizedotnetPaymentsResponse`.